detox/src/devices/drivers/android/exec/ADB.js

```diff
@@ -271,13 +271,16 @@ class ADB {
   }
 
   // TODO refactor the whole thing so as to make usage of BinaryExec -- similar to EmulatorExec
-  async adbCmd(deviceId, params, options) {
+  async adbCmd(deviceId, params, options = {}) {
     const serial = `${deviceId ? `-s ${deviceId}` : ''}`;
     const cmd = `"${this.adbBin}" ${serial} ${params}`;
-    const retries = _.get(options, 'retries', 1);
-    _.unset(options, 'retries');
+    const _options = {
+      ...options,
+      retries: options.retries || 1,
+    }
 
-    return execWithRetriesAndLogs(cmd, options, undefined, retries);
+    _.unset(options, 'retries');
```

Comment:
I don't think you longer need this `_.unset`.
AFAIR, I added it to make sure I don't pass a dirty object to `exec...` which had 5 scalar parameters at that time.

Let's remove that `_.unset` and inline that `_options` object:

```js
return execWithRetriesAndLogs(cmd, {
  retries: 1, // default value
  ...options
};
```

Reply:
🚀